### ISSUE TITLE
File Limit Request: vngrs-nlp - 500 MB

### ISSUE THREAD

**Issue by meliksahturker** [pypa/pypi-support#1917]
### Project URL

https://pypi.org/project/vngrs-nlp/

### Does this project already exist?

- [X] Yes

### New Limit

500

### Update issue title

- [X] I have updated the title.

### Which indexes

PyPI

### About the project

The project is an open source tool that implements several research papers for various NLP tasks.
Github page: https://github.com/vngrs-ai/vnlp
Documentation: https://vnlp.readthedocs.io/en/latest/

This is the first release, and it will be announced to the community once PyPi page is ready.

In order to reduce the size of the bundle:
- Deep learning models are saved in the most compact form as .hdf5 files, saving the weights only.
- No other package is bundled in the project.
- No data is bundled in the project.

### Reasons for the request

My project consists of many deep learning models.
It is these models that take the most of the space in the project.
I was unable to upload the initial release (154 MB) due to the limit.
I will release a new version very soon, including an additional deep learning model (can be up to 250 MB), hence I request an increase to 500 MB ( = 150MB + 250MB + 100 MB), in order not to ask for limit increase twice in the days to follow.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by cmaureir** [comment]
Hey @meliksahturker :wave: 
It's completely normal to have large model files in packages related to NLP, ML, etc. However I'm curious if you have explore other options instead of packaging such large files within the wheel, to provide `download` methods in your package to acquire the data, or even detecting if the current installation contains the directory where the models are. In that scenario, you could have small wheels and a more explicit way to acquire models that the users require.

**Comment by pradyunsg** [comment]
Closing this out given the lack of a response on this. Please feel welcome to come back and respond to this, or to file a new issue with the answer. :)
